kt/godot-library/build.gradle.kts

```diff
@@ -1,5 +1,18 @@
 plugins {
     kotlin("jvm")
+    id("com.utopia-rise.api-generator")
+}
+
+apply<godot.gradle.ApiGeneratorPlugin>()
+
+configure<godot.gradle.ApiGeneratorPluginExtension> {
+    outputDir.set(project.file("$rootDir/godot-library/src/main/kotlin/godot/gen"))
+    sourceJson.set(project.file("$rootDir/api-generator/src/main/resources/api.json"))
+    isNative = false
+}
+
+tasks.compileKotlin {
+    dependsOn(tasks.generateAPI)
```

Comment:
Setting the comment here as the api-generator plugin has no PR:
I would put the task `generateApi` into the group `godot-jvm` ike we do in the main gradle plugin